worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -173,8 +178,8 @@ public RegionIntersection getWalls() {
                 new CuboidRegion(pos1.withX(max.getX()), pos2.withX(max.getX())),
 
                 // Project to X-Y plane
-                new CuboidRegion(pos1.withZ(min.getZ()).add(BlockVector3.UNIT_X), pos2.withZ(min.getZ()).subtract(BlockVector3.UNIT_X)),
-                new CuboidRegion(pos1.withZ(max.getZ()).add(BlockVector3.UNIT_X), pos2.withZ(max.getZ()).subtract(BlockVector3.UNIT_X)));
+                new CuboidRegion(pos1.withZ(min.getZ()), pos2.withZ(min.getZ())),
+                new CuboidRegion(pos1.withZ(max.getZ()), pos2.withZ(max.getZ())));
```

Comment:
not sure but doesn't that change the coordinates?

Reply:
Looks like a merge mistake. I'll revert this.